### PR TITLE
add support for Mysql

### PR DESCRIPTION
**Pull request by Hebrh**
add support for Mysql #1

### CODE REVIEW

**Review by Hebrh — COMMENTED** [review]
Confirmed fully tested.

File: src/delibird/database/schema.py [hunk 1]
```diff
@@ -89,7 +114,7 @@ def create_arrow_schema(engine, dsn, table_name):
     for col in cursor.description:
         if engine == "postgresql":
             type_list.append(pa.field(col.name, arrow_type(engine, col)))
-        elif engine == "oracle":
+        elif engine == "oracle" or engine == "mysql":
```
Comment (EeroEternal) [inline src/delibird/database/schema.py:117]: oracle and mysql cursor description is same ?

Comment (Hebrh) [reply src/delibird/database/schema.py:117]: Yes! the col description of Oracle and Mysql are all tuples.

**Review by EeroEternal — APPROVED** [review]
ok

**Pull request merged**